deployments/uber-manifest.tpl

```diff
@@ -90,9 +90,13 @@ sources:
       {{ template "quay_auth" }}
       {{ end }}
       versionFilter:
-        kind: regex
+        kind: {{ (eq $id "next") | ternary "semver" "regex" }}
```

Comment:
it would be too much coupled with the current implementation - don't tie it to the version name but define a new field in the matrix e.g. `versionFilterKind`? which default to regexp for retrocompat